ts/components/conversation/Image.tsx

```diff
@@ -30,12 +30,21 @@ type Props = {
   onError?: () => void;
 };
 
+const StyledOverlay = styled.div<Pick<Props, 'darkOverlay' | 'softCorners'>>`
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  z-index: 1;
```

Comment:
Would put the z-index after left and right. This groups related css properties together.